src/evict/evict_inline.h

```diff
@@ -614,7 +614,8 @@ __evict_check_user_ok_with_eviction(WT_SESSION_IMPL *session, bool busy)
  *     should assist eviction worker threads with eviction of pages from the queues.
  *
  *     Input parameters:
- *       (1) `busy`: A flag indicating if eviction is mandatory (true) or optional (false).
+ *       (1) `busy`: A flag indicating if the session is actively pinning resources, in which
+ *            case eviction work should be limited.
```

Comment:
I find this description of the "busy" argument too concise given the amount of context behind it. With just this short explanation, it’s impossible to fully grasp its meaning and how it’s used.

I don’t think this description is sufficient, but if the goal is simply to fix a factual error in the existing text rather than improve clarity, then I’m ok with the change.

Reply:
Thanks @ershov. You're right that the goal here is mostly to correct the erroneous comment.

I've also expanded a couple other comments to add some points I mentioned in that slack thread.

If there is other stuff you think is missing, could you open a ticket for it? I'm at a disadvantage here, because I already know what this code does, so it's harder for me to see where other people have trouble following it.